clouddeploy/sso-dashboard-dev.template.yaml

```diff
@@ -23,8 +23,8 @@ spec:
   template:
     metadata:
       annotations:
-        autoscaling.knative.dev/minScale: '1'
-        autoscaling.knative.dev/maxScale: '3'
+        autoscaling.knative.dev/minScale: '0'
+        autoscaling.knative.dev/maxScale: '1'
```

Comment:
Might not be an issue in a dev context, but generally you want to allow services to scale up and back down to the desired capacity instead of down to zero and back up from there. This avoids annoying situations like waiting for services to drain connections or databases to quiesce.